src/map_extras.cpp

```diff
@@ -424,7 +425,16 @@ static void place_fumarole( map &m, const point_bub_ms &p1, const point_bub_ms &
         ignited.insert( ( i + point::south_east ) );
 
         if( one_in( 6 ) ) {
-            m.spawn_item( i + point::north_west, itype_chunk_sulfur );
+            // direction to next or previous point
+            point_rel_ms delta = i - fumarole[ index == fumarole.size() - 1 ? index - 1 : index + 1 ];
+            // rotate 90 degrees
+            std::swap( delta.raw().x, delta.raw().y );
+            if( one_in( 2 ) ) {
+                delta.raw().x *= -1;
+            } else {
+                delta.raw().y *= -1;
+            }
+            m.spawn_item( i + delta, itype_chunk_sulfur, i + delta * 2 );
```

Comment:
Previously no sulfur would spawn from a NW/SE oriented line of lava. That is fixed here by changing the base spill location here to be one tile perpendicular to the line, and the preferred fallback spill location is another tile in that direction.